api/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	DCOSVersion             string
 	SystemdUnits            []string
 
+	FlagUseTCP                     bool
```

Comment:
i removed the flag. First 3dt will try to look for available FDS and in case it couldn't find one it will use TCP port.